### ISSUE TITLE
Protecting against vulnerabilities.

### ISSUE THREAD

**Issue by iHiD** [exercism/backlog#2]
How can we check the whole of the many repos we have for vulnerabilities? Which repos do we care about? There is some discussion here: https://github.com/exercism/reboot/issues/240

**Comment by NobbZ** [comment]
That link gives a 404, is this perhaps one of the product teams private repositories?

**Comment by iHiD** [comment]
@NobbZ Yeah - that's old issue tracker from when we were making v2. I'm moving all the issues that didn't get sorted then into public repos now, but starting with "clean" issues where the person who's implementing it knows what needs doing, rather than copying dozens of old posts across.
